docs/docs-source/docs/modules/ROOT/partials/include.adoc

```diff
@@ -1,5 +1,9 @@
-:page-partial:
-// global attributes
+
+// global attributes, if you update, make sure to update in shared-content-source ROOT:partials:include.adoc too
+:supported-java-v: >= 8
```

Comment:
Global attributes can stay under this section:
https://github.com/lightbend/cloudflow/blob/master/docs/docs-source/site.yml#L23
and they will be propagated to all the docs automatically (at best of my understanding) without the need of a comment.

Reply:
I've found that authors don't tend to review the site.yml before they do a pull because it has a lot of other stuff in it. I've had better results with a separate include file. And, for some reason, I've had a problem with Antora propagating them correctly from site.yml. However, if @RayRoestenburg and @andreaTP prefer the site file. I'll test it today and make sure it works.

Reply:
I tried to move the global attributes into site.yml. However, because of the way I'm sharing the content, Antora seems to generate HTML for shared content without looking at them. So, the build is filled with errors. There is another way to share content, but it would take a lot of work and we don't know whether we will be sharing later down the road. So, I've left them in the include files for now.